server.js

```diff
@@ -1,34 +1,30 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});

```

Comment:
The mongoUri should not include the credentials and the URL should be properly formatted to establish a connection to the MongoDB database. Including sensitive information like credentials in the code is not recommended for security reasons.